lib/matplotlib/axes/_axes.py

```diff
@@ -4522,13 +4522,35 @@ def arrow(self, x, y, dx, dy, **kwargs):
         """
         Add an arrow to the axes.
 
-        Call signature::
+        Draws arrow on specified axis from (`x`, `y`) to (`x` + `dx`,
+        `y` + `dy`). Uses FancyArrow patch to construct the arrow.
 
-           arrow(x, y, dx, dy, **kwargs)
+        Parameters
+        ----------
+        x : float
+            X-coordinate of the arrow base
+        y : float
+            Y-coordinate of the arrow base
+        dx : float
+            Length of arrow in x-coordinate
+        dy : float
+            Length of arrow in y-coordinate
+
+        Returns
+        -------
+        a : FancyArrow
+            patches.FancyArrow object
+
+        Other Parameters
+        -----------------
+        kwargs : `~matplotlib.patches.FancyArrow` properties.
 
-        Draws arrow on specified axis from (*x*, *y*) to (*x* + *dx*,
-        *y* + *dy*). Uses FancyArrow patch to construct the arrow.
+        More valid arguments include:
 
+        %(FancyArrow)s : Parent object type.
```

Comment:
That's only part of it. The `FancyArrow` part also needs to be updated to match the indentation here. For me, it causes a warning which stops the build (since warnings are errors by default), though it does still look reasonable if warnings are ignored.